maintainers/maintainer-list.nix

```diff
@@ -4057,6 +4057,12 @@
     githubId = 40620903;
     name = "figsoda";
   };
+  fintohaps = {
```

Comment:
Can you introduce this change as a separate commit (still as part of this PR) please?